src/Features/Core/Portable/EditAndContinue/DebuggingSession.cs

```diff
@@ -126,7 +128,7 @@ internal EmitBaseline Test_GetProjectEmitBaseline(ProjectId id)
         {
             lock (_projectEmitBaselinesGuard)
             {
-                return _projectEmitBaselines[id];
+                return _projectEmitBaselines![id];
```

Comment:
Is the `!` a holdover from some previous changes?

Reply:
yup, good catch.